### ISSUE TITLE
Wrong lodash imports

### ISSUE THREAD

**Issue by geoynomous** [feathersjs-ecosystem/feathers-hooks-common#674]
### Steps to reproduce

Upon import seeing this

```
Cannot find module '/appserver/node_modules/.pnpm/feathers-hooks-common@6.1.0/node_modules/lodash/pick' imported from /appserver/node_modules/.pnpm/feathers-hooks-common@6.1.0/node_modules/feathers-hooks-common/dist/index.mjs
Did you mean to import lodash@4.17.21/node_modules/lodash/pick.js?
```

There is douzens of such problems with lodash

### Expected behavior

a `'.js'` seems to be missing

### Actual behavior

An error shows up

### System configuration

**Module versions** (especially the part that's not working):

6.1.1

**NodeJS version**:

16.15.0

**Operating System**:

macOS

**Comment by OADesigns** [comment]
getting same issue on Ubuntu node 16.13.1. Only when using import, works fine with require. 
Manually changing all the lodash imports in index.mjs and adding .js to end of paths also removes error.

**Comment by JoseCadenas** [comment]
I am experiencing this issue on all version 6+ updates. I am staying on 5.0.6 until a patch is found.

**Comment by fratzinger** [comment]
released as `feathers-hooks-common@6.1.5`. Please check. A confirmation that it's working would be highly appreciated
